docs/source/membership/membership.md

```diff
@@ -50,62 +94,200 @@ multiple MSPs and name them accordingly, e.g., `ORG2-MSP-NATIONAL` and
+* In channel configuration (**channel MSP**)
+
+### Local MSPs
 
-**Local MSPs are defined for clients (users) and for nodes (peers and orderers)**.
-Local MSPs define the permissions for that node (who the peer admins are, for example).
-The local MSPs of the users allow the user side to authenticate itself in its transactions
-as a member of a channel (e.g. in chaincode transactions), or as the owner of a
-specific role into the system (an org admin, for example, in configuration transactions).
+**Local MSPs are defined for clients and for nodes (peers and
+orderers)**.
+Local MSPs define the permissions for that node (who the peer admins
+are, for example). The local MSPs of clients (the account holders in
+the banking scenario above), allow the user to authenticate
+itself in its transactions as a member of a channel (e.g. in chaincode
+transactions), or as the owner of a specific role into the system (an
+org admin, for example, in configuration transactions).
 
-**Every node and user must have a local MSP defined**, as it defines who has
-administrative or participatory rights at that level (peer admins will not necessarily
-be channel admins, and vice versa).
+**Every node must have a local MSP defined**, as it defines
+who has administrative or participatory rights at that level (peer
+admins will not necessarily be channel admins, and vice versa).  This
+allows for authenticating member messages outside the context of a
+channel and to define the permissions over a particular node
+(who has the ability to install chaincode on a peer, for example).
+
+An orderer local MSP is also defined on the file system of the node
+and only applies to that node. Like peer nodes, orderers are also
+owned by a single organization and therefore have a single MSP to
+list the actors or nodes it trusts.
+
+### Channel MSPs
 
 In contrast, **channel MSPs define administrative and participatory rights at the
-channel level**. Every organization participating in a channel must have an MSP
-defined for it. Peers and orderers on a channel will all share the same view of channel
+channel level**. Peers and orderers on a channel share the same view of channel
 MSPs, and will therefore be able to correctly authenticate the channel participants.
 This means that if an organization wishes to join the channel, an MSP incorporating
 the chain of trust for the organization's members would need to be included in the
 channel configuration. Otherwise transactions originating from this organization's
-identities will be rejected.
+identities will be rejected. Whereas local MSPs are represented as a folder
+structure on the file system, channel MSPs are described in a channel
+config.json file.
+
+![MSP1d](./ChannelMSP.png)
+
+*Snippet from a channel config.json file that includes two organization
+MSPs.*
+
+**Channel MSPs identify who has permissions at a channel level**.
+
+The channel MSP defines the _relationship_ between the identities of
```

Comment:
See, I think "authority" could work here in the place of "permissions". It's not as good as "role", which implies something you're compelled to do, rather than just allowed to do, but it's a small step in the right direction.